### ISSUE TITLE
Scales for delta BC and delta L

### ISSUE THREAD

**Issue by Vasudha-pai** [RUBi-ZA/MD-TASK#53]
Hello,
I wanted to know if there is any way to normalize the scales of Y-axis in MD-TASK. I have 6 different plots with different scales, hence, I wanted to know if there is any way I can have a normalized scale for all of my 6 plots so that I can make a comparative analysis. @oliserand 
Thank you.

**Comment by Vasudha-pai** [comment]
I know we get the dat filies and we can use them to plot it. But I'm getting different plots which are not the same as delta BC :(

**Comment by oliserand** [comment]
Dear @Vasudha-pai 

I hope this reply is not too late. Can you please send let me the commands you used and the plots that you obtained? Depending on what one is wanting to do in order to compare multiple samples, the use of a common scale is fine; however if one is independently normalizing each separate run, this will only allow for a qualitative analysis, as the effects of the magnitudes across the samples are lost. Let me know if that helps, unless it is what you want to do, or I'm misunderstanding.

**Comment by Vasudha-pai** [comment]
Dear @oliserand 
Sorry for the confusion earlier. I had the files mixed up, and I apologize for any inconvenience it may have caused.
The combination of delta BC and L resulted in a total of 4 different files: Delta BC, Plus One BC, Delta L, and Standard Delta L. I had misplaced these files and used a different graph, which led to the confusion.
The Y-axis values in my files were different, making it challenging to compare them. Fortunately, using software like Origin, I was able to plot the graphs with a normalized Y axis, making interpretation easier.
I apologize for any inconvenience caused and appreciate your time and help. This issue can now be closed.
